pkg/operator/options/options.go

```diff
@@ -39,8 +39,9 @@ var (
 type optionsKey struct{}
 
 type FeatureGates struct {
-	Drift    bool
-	inputStr string
+	Drift             bool
+	inputStr          string
```

Comment:
nit: codestyle but can we throw the `inputStr` as the first member of this struct and then separate it out from the public members like

```go
type FeatureGates struct {
  inputStr string

  Drift bool
  SpotConsolidation bool
}
```

Reply:
Addressed.